### DISCUSSION TITLE
is it possible to disable this?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#1002] [Q&A]
![image](https://github.com/cedricp/ddt4all/assets/122531259/d5d81252-de6a-4fce-b249-bb581c307fe7)

**Comment by KarelSvo** [comment]
Macro Button

**Reply by DavidIgor** [reply]
in pyren ? for x67 there is not much in folder...

**Comment by AndyIreland** [comment]
should be able to disable/turn it off it your TDB cluster parameters in DDT4ALL - freeshift its called or something like that

![image](https://github.com/cedricp/ddt4all/assets/49172217/11a5ce8d-9e9b-4ff0-8acf-530319d07034)

**Reply by DavidIgor** [reply]
I have done it with pyren macros ...
